crates/bioauth-flow-rpc/Cargo.toml

```diff
@@ -15,6 +15,7 @@ rpc-error-response = { path = "../rpc-error-response" }
 rpc-validator-key-logic = { path = "../rpc-validator-key-logic" }
 
 async-trait = { workspace = true }
+futures = { workspace = true }
```

Comment:
Why do we need this?

Reply:
To use `select_next_some()`.

Reply:
https://github.com/humanode-network/humanode/pull/1018/commits/348e3683516bccbd809a5a52a99d77023b053113